src/components/Analitics/Analitics_Items/StatisticsIndividualAndTeam/IndividualStats.tsx

```diff
@@ -0,0 +1,78 @@
+import { Avatar, chakra, Flex, Text } from '@chakra-ui/react';
+
+import { AGENT_PLAN, parseDynamic } from '../../../../constants/constants';
+import { CombinedArrItem } from '../../../../contexts/StatisticsProvider';
+import CustomAnalyticsFlex from '../CustomAnaliticsFlex';
+
+export const IndividualStats = ({
+  item: {
+    fullName,
+    picture,
+    statistics: {
+      agentPrecentPlanRealization,
+      contributionToTeamPlan,
+      totalAgentChancesValue,
+      totalAgentFinalizedValue,
+    },
+  },
+}: {
+  item: CombinedArrItem;
+}) => {
+  const data = {
+    individualPlan: { label: 'individual plan: ', value: AGENT_PLAN },
+    totalSuccesses: { label: 'total successes: ', value: totalAgentFinalizedValue },
+    individualPlanRealisation: {
+      label: 'individual plan realisation: ',
+      value: agentPrecentPlanRealization,
+    },
+    contributionToTeamPlan: { label: 'contribution to team plan: ', value: contributionToTeamPlan },
+    totalChances: { label: 'total chances: ', value: totalAgentChancesValue },
+  };
+
+  const getColor = (item: { label: string; value: number | string }) => {
+    const HALF = 0.5;
+    const HUNDRED_PRECENT = 100
+    const value = parseDynamic(item.value);
+    if (item.label === data.totalSuccesses.label) {
+      if (value < AGENT_PLAN * HALF) {
+        return 'red';
+      } else if (value > AGENT_PLAN * HALF) {
+        return 'orange';
+      } else if (value > AGENT_PLAN) {
+        return 'green';
+      }
+    }
+    if (item.label === data.individualPlanRealisation.label) {
+      if (value < HUNDRED_PRECENT * HALF) {
+        return 'red';
+      } else if (value > HUNDRED_PRECENT * HALF) {
+        return 'orange';
+      } else if (value > HUNDRED_PRECENT) {
+        return 'green';
+      }
```

Comment:
ta ifologia z kolorami jest powielona więc trzeba to wyodrębnić do funkcji i nazwać jakoś np. color mapper